data/notebooks/Lab/areas_validator_test.ipynb

```diff
@@ -0,0 +1,2769 @@
+       "      <td>1_2_65</td>\n",
+       "      <td>15.410758</td>\n",
+       "      <td>2.417552</td>\n",
+       "      <td>2010</td>\n",
+       "      <td>{'units': 'ha'}</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>393</th>\n",
+       "      <td>1_2_65</td>\n",
+       "      <td>15.265364</td>\n",
+       "      <td>2.353974</td>\n",
+       "      <td>1996</td>\n",
+       "      <td>{'units': 'ha'}</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>394</th>\n",
+       "      <td>1_2_65</td>\n",
+       "      <td>15.403492</td>\n",
+       "      <td>2.353974</td>\n",
+       "      <td>2007</td>\n",
+       "      <td>{'units': 'ha'}</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>395</th>\n",
+       "      <td>1_2_65</td>\n",
+       "      <td>15.419600</td>\n",
+       "      <td>2.417552</td>\n",
+       "      <td>2016</td>\n",
+       "      <td>{'units': 'ha'}</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>396 rows × 5 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    location_id  total_area  protected_area  year         metadata\n",
+       "0        1_2_97  129.155410        1.611917  2010  {'units': 'ha'}\n",
+       "1        1_2_97  136.884844        1.725696  1996  {'units': 'ha'}\n",
+       "2        1_2_97  133.431203        1.715800  2007  {'units': 'ha'}\n",
+       "3        1_2_97  132.828293        1.687598  2016  {'units': 'ha'}\n",
+       "4        1_2_69    8.856625        4.435916  2010  {'units': 'ha'}\n",
+       "..          ...         ...             ...   ...              ...\n",
+       "391      1_2_96    2.049654        1.200173  2016  {'units': 'ha'}\n",
+       "392      1_2_65   15.410758        2.417552  2010  {'units': 'ha'}\n",
+       "393      1_2_65   15.265364        2.353974  1996  {'units': 'ha'}\n",
+       "394      1_2_65   15.403492        2.353974  2007  {'units': 'ha'}\n",
+       "395      1_2_65   15.419600        2.417552  2016  {'units': 'ha'}\n",
+       "\n",
+       "[396 rows x 5 columns]"
+      ]
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df_save = df_final_fixed.merge(locations, how ='inner',left_on='Country', right_on='name')\n",
+    "df_save = df_save.filter(['location_id', 'total_area', 'protected_area', 'year'])\n",
+    "df_save['metadata'] = str({'units':'ha'})\n",
+    "\n",
+    "df_save"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "ca0d5f16",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_save.to_csv('../../datasets/Mangrove_protected_area.csv', sep=',')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "560a5f1f",
+   "metadata": {},
+   "source": [
+    "## To-Do list  \n",
+    "- Improve validations (check unique/distinct/NA values, etc...)\n",
```

Comment:
Yes regarding this bit, i have a couple of questions that we might want to discuss in a call together.